development/nautobot_config.py

```diff
@@ -33,6 +33,7 @@
 
 PLUGINS = [
     "example_plugin",
+    "example_plugin_with_view_override",
```

Comment:
Shouldn't be enabled by default; again, it will break tests. It should only be enabled ad-hoc for demo purposes.

Reply:
Ooooh i understand now 😔